tests/containers/integration-test-local

```diff
@@ -1,8 +1,20 @@
 FROM quay.io/bluechi/integration-test-base:latest
 
+ARG with_coverage
+
 RUN mkdir -p /tmp/bluechi-rpms
+
 COPY ./bluechi-rpms /tmp/bluechi-rpms
 
+
+RUN if [ "$with_coverage" = "1" ]; then \
+        dnf install -y --repo bluechi-rpms \
+        --repofrompath bluechi-rpms,file:///tmp/bluechi-rpms/ \
+        --nogpgcheck \
+        --nodocs \
+        bluechi-coverage; \
+    fi
+
```

Comment:
Please install bluechi-coverage only after bluechi installation and before running clean all